spec/requests/api/v1/transactions/create_transfer_spec.rb

```diff
@@ -0,0 +1,90 @@
+require 'rails_helper'
+
+describe 'POST /api/organizations/#/transactions/transfer' do
+  let(:path) { "/api/organizations/#{organization.id}/transactions/transfer" }
+
+  let(:from_bank_account) { create :bank_account, organization: organization }
+  let(:to_bank_account) { create :bank_account, organization: organization }
+  let(:amount) { Money.new(10000, from_bank_account.currency) }
```

Comment:
@yunixon instead `Money.new(10000)` use `Money.from_amount(100)`. It's more readable.